policyengine_us/parameters/gov/states/la/tax/credits/child_care_expense_credit/credit_threshold.yaml

```diff
@@ -0,0 +1,11 @@
+description: Louisiana provides child care credit equals to the lesser of twenty-five dollars to individual resident whose agi is more than sixty-thousand dollars
```

Comment:
You may not want to include the specific amount in the description (e.g. $25, $6,000). You can try writing something like 'Louisiana provides child care credits up to this dollar amount...." I noticed that on the website it mentions " the credit shall be equal to the lesser of twenty-five dollars or ten percent of the federal credit for child care expenses claimed on the resident individual's federal tax return," do you think whether it would be necessary to create another parameter file for the '10%'? Also for the '$6,000' threshold?

Reply:
Thank you for your comment. Also @PavelMakarchuk  I think we will delete these parameters after the Louisiana cdcc PR is done so that I can use Yichen's code for calculating the state credit.